plugins/catalog/src/index.ts

```diff
@@ -17,7 +17,10 @@
 export * from './components/AboutCard';
 export { CatalogLayout } from './components/CatalogPage';
 export { CatalogResultListItem } from './components/CatalogResultListItem';
-export { CatalogTable } from './components/CatalogTable';
+export {
+  CatalogTable,
+  columnFactories as catalogTableColumnFactories,
```

Comment:
Let's use the pattern from [EntityTable](https://github.com/backstage/backstage/blob/809bcd396b80d032547fecac62ae9ba6f04de326/plugins/catalog-react/src/components/EntityTable/EntityTable.tsx#L77) for this instead